packages/web/src/uiUtils.ts

```diff
@@ -76,3 +78,17 @@ export function copy(text: string) {
   document.execCommand('copy');
   textArea.remove();
 }
+
+export function useSetting<S>(name: string, defaultValue: S): [S, React.Dispatch<React.SetStateAction<S>>] {
```

Comment:
I didn't find the place that imports this function.

Reply:
This is for the future use